hazelcast/src/main/java/com/hazelcast/quorum/impl/ProbabilisticQuorumFunction.java

```diff
@@ -66,34 +65,20 @@ public boolean apply(Collection<Member> members) {
             if (!isAlivePerIcmp(member)) {
                 continue;
             }
-            if (failureDetector.isAlive(member, timestamp)) {
+
+            if (isLocalDataMember(member) || failureDetector.isAlive(member, timestamp)) {
```

Comment:
We always call `apply(members)` with data members only. Only local-member check should be sufficient.